Dashboard/app/js/lib/views/data/question-answer-view.js

```diff
@@ -120,6 +124,13 @@ FLOW.QuestionAnswerView = Ember.View.extend({
     }
   }.property('this.content,this.isPhotoType,this.isVideoType'),
 
+  geoShapeObject: function(){
+    var c = this.content;
+    if (!Ember.empty(c.get('value'))) {
+      return c.get('value');
+    }
+  }.property('this.content,this.isPhotoType,this.isVideoType'),
```

Comment:
I don't quite understand why the `.property` references `isPhotoType` and `isVideoType`?